src/clarity-angular/popover/signpost/if-open.directive.specs.ts

```diff
@@ -35,9 +35,12 @@ export default function(): void {
                 expect(this.clarityDirective.open).toEqual(testValue);
             });
 
-            it("provides the current value of the open state", function() {
-                let testValue = this.clarityDirective.open;
-                expect(this.testComponent.openState).toBe(testValue);
+            it("gets the current value of the open state", function() {
+                let testValue: boolean = this.ifOpenService.open;
+
+                expect(this.ifOpenService.open).toEqual(this.clarityDirective.open);
+                this.ifOpenService.open = !testValue;
+                expect(this.ifOpenService.open).toEqual(this.clarityDirective.open);
```

Comment:
What you want here is to test the two-way binding, right?

So you should have the de-sugarized syntax `<ng-template [(clrIfOpen)]="openState">` in your test component, and then check the `testComponent` itself:
```typescript
this.ifOpenService.open = true;
expect(this.testComponent. openState).toEqual(true);
this.ifOpenService.open = false;
expect(this.testComponent. openState).toEqual(false);
```